src/elements/OcDatepicker.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <datetime
+    v-model="update"
+    class="oc-datepicker"
+    :title="title"
+    :type="type"
+    :phrases="phrases"
+    :min-datetime="minDatetime"
+    :max-datetime="maxDatetime"
+    @cancel="$_cancel()"
+    @input="$_input()"
+  ></datetime>
+</template>
+
+<script>
+import { Datetime } from "vue-datetime"
+import moment from "moment"
+
+/**
+ * Example component is used to visually communicate core parts of the product
+ * and available actions.
+ */
+export default {
+  /**
+   * Component names should be short, pronounceable and Capitalized.
+   */
+  name: "oc-datepicker",
+  /**
+   * Components in the system are labelled with status labels that reflect their
+   * state of completion. See example below. All available statuses are:
+   *
+   * STATUS:        COLOR:     DESCRIPTION:
+   *
+   * deprecated     Red        Component is deprecated
+   * prototype      Blue       Prototype, do not implement!
+   * under-review   Yellow     Component is currently being reviewed
+   * ready          Green      Ready to be used
+   */
+  status: "prototype",
+  /**
+   * Release indicates when this component was added into the system.
+   * (in which design system version)
+   */
+  release: "1.0.0",
+  /**
+   * Prop definitions should be as detailed as possible, specifying at least
+   * type(s). See examples below:
+   */
+  props: {
+    /**
+     * The html element name used for the container of Example component.
+     */
+    date: {
+      type: String,
+      default: moment().format(),
+    },
+    /**
+     * Popup title.
+     */
+    title: {
+      type: [Boolean, String],
+      default: false,
+      required: false,
+    },
+    /**
+     * Picker type.
+     * `date, datetime, time`
+     */
+    type: {
+      type: String,
+      default: "date",
+      validator: date => {
+        return date.match(/(date|datetime|time)/)
+      },
+    },
+    phrases: {
+      type: Object,
+      default: () => {
+        return { ok: "Ok", cancel: "Cancel" }
+      },
+      validator: value => {
+        return typeof value.ok === "string" || typeof value.cancel === "string"
+      },
+    },
+    /**
+     * Minimum datetime.
+     * `ISO 8601 String`
+     */
+    minDatetime: {
+      type: [Object, String],
```

Comment:
Object?

Reply:
`null` is an object type. vue-datetime expects a string or null.